client/landing/site.landing/coffee/main.coffee

```diff
@@ -37,7 +37,7 @@ do ->
     new MainController group
 
   kd.config             or= {}
-  kd.config.environment   = if location.hostname is 'koding.com' then 'production' else 'development'
+  kd.config.environment   = window._runtimeOptions.environment
```

Comment:
we should either go safe and place a landmine such as
`kd.config.environment  = window._runtimeOptions.environment or 'development'`
or we do `throw  unless kd.config.environment` to be safer since this is an important piece.